server/channels/store/sqlstore/user_store.go

```diff
@@ -2140,7 +2140,11 @@ func (us SqlUserStore) IsEmpty(excludeBots bool) (bool, error) {
 		From("Users")
 
 	if excludeBots {
-		builder = builder.LeftJoin("Bots ON Users.Id = Bots.UserId").Where("Bots.UserId IS NULL")
+		if isPostgreSQL := us.DriverName() == model.DatabaseDriverPostgres; isPostgreSQL {
+			builder = builder.LeftJoin("Bots ON Users.Id = Bots.UserId").Where("Bots.UserId IS NULL")
+		} else {
+			builder = builder.Where(sq.Expr("Users.Id NOT IN (SELECT UserId FROM Bots)"))
```

Comment:
0/5, `Users.Id` can just be simplified to `Id`

Reply:
Should we actually aim to be explicit about tables? It's not an issue in this case, but this class of problem is what caused https://github.com/mattermost/mattermost/pull/20074, and I wonder if we'd be better being consistently verbose instead of saving a few characters?

Reply:
+1 to using the table name